test/unit/module/module.spec.js

```diff
@@ -21,9 +21,156 @@ describe('Module', () => {
   })
 
   it('get namespacer: namespace option is true', () => {
-    const module = new Module({
+    let module = new Module({
       namespaced: true
     })
     expect(module.namespaced).toBe(true)
+
+    module = new Module({
+      namespaced: 100
+    })
+    expect(module.namespaced).toBe(true)
+  })
+
+  it('add child method', () => {
+    const module = new Module({})
+
+    module.addChild('v1', new Module({}))
+    module.addChild('v2', new Module({}))
+    expect(Object.keys(module._children)).toEqual(['v1', 'v2'])
+  })
+
+  it('remove child method', () => {
+    const module = new Module({})
+
+    module.addChild('v1', new Module({}))
+    module.addChild('v2', new Module({}))
+    module.removeChild('v2')
```

Comment:
We should also check the `_children` just before calling `removeChild` to confirm whether it actually works or not.